server/controllers/peopleController.ts

```diff
@@ -0,0 +1,17 @@
+import type { Request, RequestHandler, Response } from 'express'
+import paths from '../paths/apply'
+
+export default class PeopleController {
+  find(): RequestHandler {
+    return async (req: Request, res: Response) => {
+      const { crn } = req.body
+
+      if (crn) {
+        // TODO: error handling if invalid/not found CRN
```

Comment:
One thing worth pointing out (and something we need to handle in some cases) is if the user can't access the CRN (it's not it their caseload, or it's a restricted access offender (usually someone famous etc), the 403 response will bubble up from the API and log the user our.